set_env_variables.sh

```diff
@@ -8,8 +8,22 @@ log() {
   echo -e "${COLOR_CYAN}$1${COLOR_DEFAULT}"
 }
 
-if [ -z ${SCRIPT_DIR+x} ]; then
-  SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# if the script is run with Jenkins, then set the SCRIPT_DIR manually, otherwise use the old way (SET JENKINS TO 1)
+#if [[ -z "${JENKINS}" ]]; then
+#   not run with JENKINS
+#  if [ -z ${SCRIPT_DIR+x} ]; then
+#    SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+#  fi
+#else
+  # run with JENKINS, need to set the SCRIPT_DIR in a different way, more manually
+#  SCRIPT_DIR=`pwd`/ampere_model_library
+#fi
+
+# checks if IS_JENKINS has length equal to zero
+if [[ -z "${IS_JENKINS}" ]]; then
+  SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
What if it's not "jenkins", but script_dir is set (e.g. by model_zoo_dev)?

Reply:
Actually given that CI uses model_zoo_dev, instead of IS_JENKINS variable, this should be solved by passing a proper value (I assume pwd/ampere_model_library) from the level of model_zoo_dev wrapper (setup.sh -> https://github.com/AmpereComputingAI/model_zoo_dev/blob/main/setup.sh). That's the reason SCRIPT_DIR variable was here in the first place.